File: Csharp/URI/01-Iniciantes/01-nivel/1006.cs
using System; 
namespace Exe1006.URI
{
    class Exe1006
    {
        public  Exe1006() 
        { 
            double firstGrade, secondGrade , thirdGrade, averageGrade;
            Console.WriteLine("Digite a primeira nota");
            firstGrade = double.Parse(Console.ReadLine());
            Console.WriteLine("Digite a segunda nota");
            secondGrade = double.Parse(Console.ReadLine());
            Console.WriteLine("Digite a terceira nota");
            thirdGrade = double.Parse(Console.ReadLine());
            averageGrade = ((firstGrade * 2) + (secondGrade * 3) + (thirdGrade * 5))/ 10.0;
            Console.WriteLine("MEDIA = {0:0.0}", averageGrade);
        }
    }
}

File: Javascript/URI/01-Iniciantes/1000-1020/1006.js
let firstGrade, secondGrade, thirdGrade , averageGrade;

firstGrade = parseFloat(prompt("Digite a primeira nota"));
secondGrade =  parseFloat(prompt("Digite a segunda nota"));
thirdGrade =  parseFloat(prompt("Digite a terceira nota"));
averageGrade = ((firstGrade * 2) + (secondGrade * 3) + (thirdGrade * 5))/ 10.0;
console.log(`MEDIA = ${averageGrade.toFixed(1)}`);
alert(`MEDIA = ${averageGrade.toFixed(1)}`);